src/main/java/com/synopsys/integration/detect/configuration/DetectConfigurationFactory.java

```diff
@@ -463,4 +445,22 @@ public ProjectGroupOptions createProjectGroupOptions() {
         String projectGroupName = detectConfiguration.getNullableValue(DetectProperties.DETECT_PROJECT_GROUP_NAME);
         return new ProjectGroupOptions(projectGroupName);
     }
+
+    public List<String> collectSignatureScannerDirectoryExclusions() {
+        return collectDirectoryExclusions(DefaultSignatureScannerExcludedDirectories.getDirectoryNames());
+    }
+
+    private List<String> collectDirectoryExclusions() {
+        return collectDirectoryExclusions(Collections.emptyList());
+    }
+    private List<String> collectDirectoryExclusions(@NotNull List<String> givenExclusions) {
+        List<String> directoryExclusionPatterns = new ArrayList<>(detectConfiguration.getValue(DetectProperties.DETECT_EXCLUDED_DIRECTORIES));
+
+        if (Boolean.FALSE.equals(detectConfiguration.getValue(DetectProperties.DETECT_EXCLUDED_DIRECTORIES_DEFAULTS_DISABLED))) {
```

Comment:
What are your thoughts on changing `DETECT_EXCLUDED_DIRECTORIES_DEFAULTS_DISABLED=false` to `DETECT_DEFAULT_DIRECTORIES_EXCLUDED=true`?

Reply:
I think the only reason we went with that horrible name is because it then shared a prefix "detect.excluded.directories" with the related properties.  I'm not sure if that validates the eyesore